README.md

```diff
@@ -61,23 +54,36 @@ arbitrary = { version = "1.0", optional = true } # Enable with `arb` feature
 ```
 
 To use the code, add `mod messages` to your `lib.rs` (or `main.rs`).
-You will most likely want to interact with the generated
-`Messages` enum, and call `Messages::from_can_message(id, &payload)`.
+You will most likely want to interact with the generated `Messages` enum, and call `Messages::from_can_message(id, &payload)`.
 
-Note: The generated code contains a lot of documentation!
-Give `cargo doc --open` a try.
+Note: The generated code contains a lot of documentation.
+Give it a try:
+```bash
+cargo doc --open
+```
 
 ### Feature flags
 
 The following (optional) features can be specified:
 
-- `debug`: enables `Debug` derive
+- `debug`: enables `#[derive(Debug)` for messages
 - `range_checked`: adds range checks in setters
-- `arb`: Enables implementation of [`Arbitrary`] trait.
-  Also requires you add `arbitrary` crate (version 1.x) as a dependency of the feature,
-  using `arb = ["arbitrary"]`.
+- `arb`: enables implementation of [`Arbitrary`] trait.
+  Also requires you to add `arbitrary` crate (version 1.x) as a dependency of the feature, using `arb = ["arbitrary"]`.
+  [`Arbitrary`]: https://docs.rs/arbitrary/1.0.0/arbitrary/trait.Arbitrary.html
+- `std`: Implements `std::error::Error` for `CanError`. This makes it easy to use `anyhow` for error handling.
+
+To enable all features add this to your `Cargo.toml`:
 
-[`Arbitrary`]: https://docs.rs/arbitrary/1.0.0/arbitrary/trait.Arbitrary.html
+```toml
+# features for dbc-codegen `messages.rs` file
+[features]
+default = ["debug", "arb", "range_checked", "std"]
```

Comment:
I think I would move "debug", "arb", "range_checked" to std. Whether to enable std by default or not, not sure, probably @killercup should decide.

Reply:
Let's have a discussion after this lands :)

Reply:
Yes, from my side now everything should be ok and this can be merged.